src/op.c

```diff
@@ -1,9 +1,9 @@
-#include "corewar.h"
+#include "../include/op.h"  // op_t
```

Comment:
It's not necessary to include the whole file path to the header file. My Makefile is set up so that you can just do `#include "op.h"`.